server/form-pages/apply/basic-information/situation.ts

```diff
@@ -1,24 +1,37 @@
+import { SessionDataError } from '../../../utils/errors'
+import { retrieveQuestionResponseFromSession } from '../../../utils/utils'
 import TasklistPage from '../../tasklistPage'
+import { SentenceTypesT } from './sentenceType'
 
-// TODO: The options that appear will depend on the response to the previous question.
-// For now we'll return all options
-export enum Situations {
-  riskManagement = 'Referral for risk management',
-  residencyManagenent = 'Residency management',
-  bailAssessment = 'Bail assessment for community penalty',
-  bailSentence = 'Bail sentence',
-}
+const communityOrderSituations = {
+  riskManagement: 'Referral for risk management',
+  residencyManagement: 'Residency management',
+} as const
+type CommunityOrderSituations = typeof communityOrderSituations
+
+const bailPlacementSituations = {
+  bailAssessment: 'Bail assessment for community penalty',
+  bailSentence: 'Bail sentence',
+} as const
+type BailPlacementSituations = typeof bailPlacementSituations
```

Comment:
I wonder here if if might be clearer to have `const situations` and then use the `Pick` utility type to pick the ones you want for each situation (as you have in the next commit)